Tests/iaas/image-metadata/image-md-check.py

```diff
@@ -230,18 +231,21 @@ def main(argv):
             cloud = opt[1]
     images = args
     if not cloud:
-        print("ERROR: Need to specify --os-cloud or set OS_CLOUD environment.", file=sys.stderr)
+        print("CRITICAL: Need to specify --os-cloud or set OS_CLOUD environment.", file=sys.stderr)
         usage(1)
-    conn = openstack.connect(cloud=cloud, timeout=24)
-    # Do work
-    if not images:
-        images = get_imagelist(private)
-    err = 0
-    # Analyse image metadata
-    for image in images:
-        err += validate_imageMD(image)
-    if not skip:
-        err += report_stdimage_coverage(images)
+    try:
+        conn = openstack.connect(cloud=cloud, timeout=24)
```

Comment:
The timeout should be configurable. Or we should use 42 instead of 24 ;)

Reply:
I haven't touched the timeout part. I will gladly change it to 42...

Reply:
If this is not from you, I would change it later. Both 24 and 42 looks random. I don't know why a timeout is necessary here. I would expect the API to respond directly.